content/rosa/external-auth/index.md

```diff
@@ -0,0 +1,545 @@
+
+Click on the "Token configuration" sub-blade and select the "Add optional claim" button.
+
+![Azure Portal - Add Optional Claims Page](./images/azure-portal_optional-claims-page.png)
+
+Select ID then check the "email" and "preferred_username" claims and click the "Add" button to configure them for your Azure AD application.
+
+![Azure Portal - Add Optional Claims - Token Type](./images/azure-portal_add-optional-claims-page.png)
+![Azure Portal - Add Optional Claims - email](./images/azure-portal_add-optional-email-claims-page.png)
+![Azure Portal - Add Optional Claims - preferred_username](./images/azure-portal_add-optional-preferred_username-claims-page.png)
+
+When prompted, follow the prompt to enable the necessary Microsoft Graph permissions.
+
+![Azure Portal - Add Optional Claims - Graph Permissions Prompt](./images/azure-portal_add-optional-claims-graph-permissions-prompt.png)
+
+Add group claims, make sure to select groups assigned to the application.
+
+![Azure Portal - Add Optional Claims - Graph Permissions Prompt](./images/azure-portal_add-optional-claims-group.png)
+
+At the time of this writing, the --claim-mapping-username-claim must be 'email' and the --claim-mapping-groups-claim will be 'groups'.
+
+![Azure Portal - Add Optional Claims - Graph Permissions Prompt](./images/azure-portal_add-optional-claims-token-config.png)
+
+From the above example, these are variables we will use next.
+
+| <b>Flag    | Description<b>  | 
+| :--------- | :---------------------------|
+| --cluster | kmc-external | 
+| --name | kmc-external-auth
+| --issuer-url | https://login.microsoftonline.com/tenant_id_xxxxxx/v2.0 |  
+| --issuer-audiences | Application ID from Step 1 in Register a new application in Azure Entra ID | 
+| --claim-mapping-username-claim | email | 
+| --claim-mapping-groups-claim | groups | 
+| --console-client-id | Application ID from Step 1 in Register a new application in Azure Entra ID | 
+| --console-client-secret | Value from Step 1 in Register a new application in Azure AD |
+
+### Set Environment Variables (values from entraid application)
+```bash
+IDP_NAME=hcp-extauth
+CLIENT_ID=<application_id>
+CLIENT_SECRET=<client_secret_value>
+TENANT_ID=<tenant_id>
+API_URL=$(rosa describe cluster --cluster $ROSA_CLUSTER_NAME -o json | jq -r '.api.url')
+```
+
+### Create the ROSA External Auth Provider
+
+```bash
+rosa create external-auth-provider --cluster=${ROSA_CLUSTER_NAME} --name=${IDP_NAME} --issuer-url=https://login.microsofton
+line.com/${TENANT_ID}/v2.0 --issuer-audiences=${CLIENT_ID} --claim-mapping-username-claim=email --claim-mapping-groups-claim=groups   --console-client-id=${CLIENT_ID} --console-client-secret=${CLIENT_SECRET}
+```
+Once the External Authentication Provider has been created, it will take several minutes for the provider to become active on the ROSA cluster
```

Comment:
```suggestion
Once the External Authentication Provider has been created, it will take several minutes for the provider to become active on the ROSA cluster.
```